hazelcast/src/test/java/testsubjects/ClientOnlyStaticSerializableBiFunction.java

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
From here down, its all test classes I wrote in non-hazelcast base package. 
These are to make sure that the solution works for static/non-static and serializable/non-serializable bi-functions